lib/helpers/getBasicDomNodeProfile.js

```diff
@@ -14,24 +14,29 @@ function getAttr(el, name) {
 }
 
 export default function getBasicDomNodeProfile(el) {
+  let all_attrs = getDomNodeAttributes(el, EXCLUDE_VALUE_REGEX);
```

Comment:
Maybe this is a dumb question, but does this need to be a copy, rather than the original?

Reply:
Good question: `getDomNodeAttributes` is computing a literal value, not accessing & returning a reference to something, it shouldn't matter here 👍🏻

Objects in JS are indeed mutable, but we're not closing over anything in this case: `getDomNodeAttributes` returns an object literal, and then we're just shoving it into another object literal, and then serializing it later on. They're all intermediate objects, and nothing is maintaining a reference to them: they'll fall out of scope each time the function call frame is popped off the stack.